tutoraspects/templates/aspects/apps/aspects/migrations/README.rst

```diff
@@ -0,0 +1,34 @@
+Alembic migrations for Aspects
+===============================
+
+Alembic is a migration tool for SQLAlchemy. It is used to manage the clickhouse database
+schema for the Aspects project.
+
+For more information about Alembic, see https://alembic.sqlalchemy.org/en/latest/
+
+Aspects interface for Alembic
+-----------------------------
+
+The Aspects project provides a custom interface for Alembic. This interface is
+available in the command line with the subcommand ``tutor local|dev alembic do -c "custom command"``.
+
+For example, to run the command ``alembic upgrade head`` in the dev environment, you would run::
+
+    tutor dev alembic do -c "upgrade head"
```

Comment:
For all of these commands I had to do them in a different order: `tutor local do alembic -c "upgrade head"`